src/main/java/com/dougnoel/sentinel/webdrivers/Driver.java

```diff
@@ -45,6 +45,13 @@ public static WebDriver getWebDriver() {
     	return getSentinelDriver().getWebDriver();
     }
 
+	/**
+	 *  Sets the session's webdriver to the new webdriver passed.
+	 */
+	public static void setWebDriver(WebDriver newDriver){
```

Comment:
Delete this, I don't want to ever allow people to fuck with the underlying driver.